app/json_util.py

```diff
@@ -52,5 +52,5 @@ def default(self, o):
         return json.JSONEncoder.default(self, o)
 
 
-def to_json(obj):
-    return json.dumps(obj, cls=Encoder, indent=2, ignore_nan=True)
+def to_json(obj, **kwargs):
```

Comment:
Pop indent from kwargs with 2 as the default, if it is not found. The rest looks good.

Reply:
Yes added